app/controllers/campaign_pages_controller.rb

```diff
@@ -35,13 +35,13 @@ def show
     @template = Liquid::Template.parse(markup)
 
     @data = Plugins.data_for_view(@campaign_page).
-      merge( @campaign_page.attributes ).
+      merge( @campaign_page.liquid_data ).
       merge( 'images' => images ).
       merge( LiquidHelper.globals ).
       merge( 'shares' => Shares.get_all(@campaign_page) ).
       deep_stringify_keys
 
-    render :show, layout: 'liquid'
+    render :show, layout: 'sumofus'
```

Comment:
Maybe this could be set in `config/champaign.yml`, to avoid it being hardwired in.